conans/client/graph/graph_manager.py

```diff
@@ -199,8 +200,8 @@ def _load_root_direct_reference(self, reference, graph_lock, profile):
         conanfile = self._loader.load_virtual([reference], profile)
         root_node = Node(ref=None, conanfile=conanfile, context=CONTEXT_HOST, recipe=RECIPE_VIRTUAL)
         if graph_lock:  # Find the Node ID in the lock of current root
-            node_id = graph_lock.get_node(reference)
-            locked_ref = graph_lock.pref(node_id).ref
+            node_id = graph_lock.get_node_by_req(reference)
```

Comment:
Is this intended or a typo?

```suggestion
            node_id = graph_lock.get_node_by_ref(reference)
```